stubs/pygit2/pygit2/config.pyi

```diff
@@ -0,0 +1,60 @@
+from _typeshed import AnyStr_co
+from os import PathLike
+from typing_extensions import Self
+
+from _cffi_backend import _CDataBase
+
+def str_to_bytes(value: object, name: str) -> bytes: ...
```

Comment:
```suggestion
def str_to_bytes(value: str, name: object) -> bytes: ...
```
It throws a TypeError if it's not a str

Reply:
That must be a thinko. Fixed in the next push.